### ISSUE TITLE
New Task: Fix bug in mermaid diagram

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/marine_life_data_network#99]
### Who is requesting this?

@mathewbiddle

### What is being requested?

Mermaid diagrams aren't displaying appropriately on the website. https://ioos.github.io/marine_life_data_network/mldn-data-flow.html

![image](https://github.com/user-attachments/assets/683f9fd6-6b1d-46a5-af7e-98bf1e4ab2be)

### What is the requested deadline and why?

ASAP

### What is the current status quo (i.e., what happens if this does not get done)?

Diagrams do not display correctly. Community has no clue what an appropriate data flow should be for marine life data.

### What indicates this is done (i.e., how do we know this is complete)?

- [x] mermaid is displayed correctly
- [x] PR merged to update webpage

### Provide a description or any other important information.

As a short term solution, we could use a previous trick to get the diagrams to display. See https://github.com/ioos/mbon-docs/blob/bc5a55ac8dbff9133a903b0d9703957295658cd7/_docs/mbon-data-flow.md?plain=1#L12-L13

This is a hack, but it will work in a pinch.

**Comment by MathewBiddle** [comment]
Also, it looks like the TOC is all messed up too:

![image](https://github.com/user-attachments/assets/328bfac9-0ee8-4590-b4e2-4180fa9624dc)

**Comment by conniekot** [comment]
@MathewBiddle Can you please change this Status = Done? I can't do that - thanks!

**Comment by MathewBiddle** [comment]
I was waiting on moving things to Done until our monthly Marine Life meeting with Axiom. There are a few items in the `In Progress` column that are `Done`, but I want everyone to be in agreement that the task is `Done`.

**Comment by MathewBiddle** [comment]
Yup, this is `Done`.
